frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -202,6 +202,12 @@ pods:
                             -f ./apache-cassandra-{{CASSANDRA_VERSION}}/conf/cassandra.yaml \
                             -d ${LOCAL_SEEDS} \
                             -p ${CASSANDRA_NATIVE_TRANSPORT_PORT} "$t" ;
+                        result=$?
+                        if [ $result != 0 ]
```

Comment:
could do `if [ $result != 0 ]; then` to match with the for loop formatting above

Reply:
(we've done it everyone, we've reached peak nitpick)